packages/app-staking/src/Account/index.tsx

```diff
@@ -118,6 +121,19 @@ class Account extends React.PureComponent<Props, State> {
     );
   }
 
+  private renderUnbond () {
+    const { controllerId } = this.props;
+    const { isUnbondOpen } = this.state;
+
+    return (
+        <Unbond
+          controllerId={controllerId}
+          isOpen={isUnbondOpen}
```

Comment:
There's  no need to pass`isOpen` down to the component. You can just conditionally render  `<Unbond>` it here in the index.tsx (see comment above)